### ISSUE TITLE
All game freeze after 5-10 minutes

### ISSUE THREAD

**Issue by crash9902** [RPCS3/rpcs3#9144]
## Quick summary

Games freeze after 5-10 minutes of playing

## Details

I updated my emulator to newest version and then I launch Demon's souls and started to play. Then after 5-10 minutes my game freeze and nothing happened. I reload game and after 2 minutes my game freeze again. Then I tried all the methods to do something, but nothing helped, the game just freezes. In other games - same situation. 
But when I launch from backup old version of emulator 0.0.10-10321-72bed8dd alfa - everything was fine

#### Please provide _exact_ build (or commit) information that introduced the regression you're reporting.

Newest build 0.0.12-11088-8307f82f alpha

#### TWO logs:
* One for the build with regression.
[RPCS3.log](https://drive.google.com/file/d/1j3Tct4u98uHWXtm4SQMEkZbtvyFf-cZZ/view?usp=sharing)
* One for the build that works as expected.
[RPCS3.log from version 0.0.10-10321-72bed8dd alfa](https://drive.google.com/file/d/1ol6MPe3PTt2jb5r7RJon-EDm2RgtBc7n/view?usp=sharing)

#### My system configuration:
* OS Windows 7 max x64
* CPU Intel Core i7-3770
* GPU Nvidia 1060
* Driver version 456.71

**Comment by AniLeo** [comment]
> Please provide exact build (or commit) information that introduced the regression you're reporting.

> Newest build 0.0.12-11088-8307f82f alpha

False. This only changes something in CI, therefore that's impossible.
If you don't know, don't give incorrect information, just say you don't know.

Missing version that introduced this regression.

**Comment by havefunrecs** [comment]
I have the same build (RPCS3 v0.0.12-11088-8307f82f Alpha) and am having the same issue.

**Comment by AniLeo** [comment]
Ok, but we need to know what build introduced the regression

**Comment by RainbowCookie32** [comment]
Left Demons Souls running for 30 minutes earlier today and it didn't freeze. 
Tested on Manjaro with a 1700 and RX 570

**Comment by Asinin3** [comment]
@havefunrecs Then determine where the regression started.. Use https://rpcs3.net/compatibility?b
check old builds. Keep testing until you find which build the issue started happening in. Vague issue reports where we have no idea where the issue started happening are not helpful. 

When you create a regression issue report on github, it asks a lot of questions and explains how to do a proper report: https://github.com/RPCS3/rpcs3/issues/new?assignees=&labels=&template=1-regression-report.md&title= read through that and then make a new post here with more info.

**Comment by shattenjager** [comment]
> Ok, but we need to know what build introduced the regression

I just updated to 0.0.12-11088 and I had the same problem with the freeze. I reverted back to 0.0.12-11068 and game doesn't freeze anymore. I'm playing Demon's Souls.

**Comment by havefunrecs** [comment]
I reverted back to 0.0.12-11068 as well and it's working for me. I'm playing NCAA Football 14.

**Comment by RainbowCookie32** [comment]
> > Ok, but we need to know what build introduced the regression
> 
> I just updated to 0.0.12-11088 and I had the same problem with the freeze. I reverted back to 0.0.12-11068 and game doesn't freeze anymore. I'm playing Demon's Souls.

There are 10 builds in that range. We need to track it down to the specific build that broke it

**Comment by Parotaku** [comment]
Maybe I can help with that:
For me, the freezes/crashes started with 0.0.12-11077...
'Puppeteer' (BCES00935) seems to be very sensitive to this issue as it freezes almost immediatly, before even reaching the title/menu... reverting to the previous build (0.0.12-11069) made the freezes/crashes stop...

**Comment by CoolEmuGuy** [comment]
I can confirm version 0.0.12-11077 causes the crashes. it happened to me too, even on the latest version. I have the same spec as TS.

**Comment by griever-gf** [comment]
Looks like #9238, try to off TSX instructions.

**Comment by OrdinaryMagician** [comment]
May not be related, I'm on a CPU without TSX and I also get these freezes.

**Comment by Asinin3** [comment]
Still an issue?

**Comment by mrtawsiti3** [comment]
> Still an issue?

Yes happens to me aswell, playing game 1 - 10 mins and then random screen freeze, sound continues to go. not all of the games are affected but most of them are, games affected i tested: TLOU Vulkan, NBA 2K17 OpenGL and Vulkan, GOW Ascension Vulkan, Uncharted 1,2,3 Vulkan, Killzone 3 Vulkan...
 
My Log and TTY while playing TLOU for 10sec:
https://www.dropbox.com/s/s3qgzmswlmr0ecv/log%20tty.rar?dl=0

Specs:
CPU:   AMD Ryzen 7 5800X 8 Core Processor 3.80 GHz 7nm
GPU:   NVIDIA GeForce RTX 3060 Ti 8GB 8nm
MBO:  ASRock B550M Steel Legend (AM4)
RAM:  G.SKILL Ripjaws 2x16 GB 3600 MHz CL16
SSD:   Samsung 970 EVO 500GB NVMe M.2
SSD:   Samsung 970 EVO 2TB NVMe M.2
PSU:  Corsair TX550M 80 + GOLD 550W
CPU Cooler:  Noctua NH-U12S
CASE:  Lian Li LANCOOL II Mesh RGB
OS:  Windows 10 Pro 64-Bit

Emulator: RPCS3 0.0.15 - 11882

**Comment by Daitan9** [comment]
13/04/2021,
Same problem with the latest version : 0.0.15-12082-006b8574 Alpha
The bug occurs on TLOU.

**Comment by Megamouse** [comment]
TSX stuff was changed recently. Please retest.

**Comment by Megamouse** [comment]
Closing due to inactivity

**Comment by saeed-neorelm** [comment]
this happened to me with the new updated
